exporter/metrics_exporter.go

```diff
@@ -67,24 +67,33 @@ func newMetricsExporter(
 }
 
 func (m *metricsExporter) run(ctx context.Context) {
```

Comment:
Only change this PR needs for me to approve-- change `run` to `start`, please. It's a subtle distinction, but `run` tends to imply the function will block the current goroutine and should therefore probably be invoked with the `go` keyword. This function does _not_ block the current goroutine. (It starts a bunch of them and then returns almost immediately.) The function name `start` conveys that this begins a process and then returns.